### PR TITLE
MC-16275: Add documentation for the changes to the get billing cycle …

### PR DESCRIPTION
**Pull request by HassanHaidar**
…endpoint

#### Changes made

- Add documentation for the endpoint changed (get billing cycle)

### CODE REVIEW

File: source/includes/administration/_organizations.md [hunk 1]
```diff
@@ -974,4 +974,80 @@ curl "https://cloudmc_endpoint/api/v1/organizations/87895f43-51c1-43cc-b987-7e30
...
+
+<!-------------------- FIND BILLING CYCLE -------------------->
+### Get a billing cycle for an org
+
+`GET /organizations/:organization_id/billing_cycles/:billing_cycle_id`
+
+Retrieves the billing cycle for the organization.
+
+```shell
+# Retrieve billing emails
```
Comment (fyrnaga) [inline source/includes/administration/_organizations.md]: seems not right as comment

File: source/includes/administration/_organizations.md [hunk 2]
```diff
@@ -974,4 +974,80 @@ curl "https://cloudmc_endpoint/api/v1/organizations/87895f43-51c1-43cc-b987-7e30
...
+
+Attributes | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The id of the billing cycle.
+`startDate`<br/>*string* | The start date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) of the billing cyle.
+`endDate`<br/>*string* | The end date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) of the billing cyle.
+`currency`<br/>*string* | The currency associated to the organization.
+`deferred`<br/>*boolean* | Indicated whether the billing cycle is deferred or not.
+`invoices`<br/>*Array[[Invoice](#administration-invoices)]* | List of invoices associated with the billing cycle.
+`pricingPackages.endDate`<br/>*string* | The start date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) of when the pricing package was applied to the billing cycle.
```
Comment (fyrnaga) [inline source/includes/administration/_organizations.md]: this is not the start date but the end date

File: source/includes/administration/_organizations.md [hunk 3]
```diff
@@ -974,4 +974,80 @@ curl "https://cloudmc_endpoint/api/v1/organizations/87895f43-51c1-43cc-b987-7e30
...
+`pricingPackages.startDate`<br/>*string* | The end date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) of the pricing package associated with the billing cycle.
```
Comment (fyrnaga) [inline source/includes/administration/_organizations.md]: seems it was inverted

File: source/includes/administration/_organizations.md [hunk 4]
```diff
@@ -974,4 +974,80 @@ curl "https://cloudmc_endpoint/api/v1/organizations/87895f43-51c1-43cc-b987-7e30
...
+`pricingPackages.name`<br/>*Map[String, String]* | The name translations of the pricing package.
+`pricingPackages`<br/>*Array[BillingCycle.BillingCyclePackage]* | List of pricing packages associated with the billing cycle.
```
Comment (fyrnaga) [inline source/includes/administration/_organizations.md]: This should be before the attributes of it.

File: source/includes/administration/_organizations.md (hunk 1 above)
Comment (HassanHaidar) [reply source/includes/administration/_organizations.md]: 56959e238bd33b06a583d71a44d5299e10d8bbc1

File: source/includes/administration/_organizations.md [hunk 5]
```diff
@@ -974,4 +974,80 @@ curl "https://cloudmc_endpoint/api/v1/organizations/87895f43-51c1-43cc-b987-7e30
...
+```
+
+Attributes | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The id of the billing cycle.
+`startDate`<br/>*string* | The start date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) of the billing cyle.
+`endDate`<br/>*string* | The end date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) of the billing cyle.
+`currency`<br/>*string* | The currency associated to the organization.
+`deferred`<br/>*boolean* | Indicated whether the billing cycle is deferred or not.
+`invoices`<br/>*Array[[Invoice](#administration-invoices)]* | List of invoices associated with the billing cycle.
```
Comment (GuillaumeVialle) [inline source/includes/administration/_organizations.md]: For the type, I think we generally use Array[Object]

File: source/includes/administration/_organizations.md [hunk 6]
```diff
@@ -974,4 +974,80 @@ curl "https://cloudmc_endpoint/api/v1/organizations/87895f43-51c1-43cc-b987-7e30
...
+`pricingPackages.endDate`<br/>*string* | The start date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) of when the pricing package was applied to the billing cycle.
+`pricingPackages.startDate`<br/>*string* | The end date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) of the pricing package associated with the billing cycle.
+`pricingPackages.name`<br/>*Map[String, String]* | The name translations of the pricing package.
+`pricingPackages`<br/>*Array[BillingCycle.BillingCyclePackage]* | List of pricing packages associated with the billing cycle.
```
Comment (GuillaumeVialle) [inline source/includes/administration/_organizations.md]: same here, Array[Object]

File: source/includes/administration/_organizations.md [hunk 7]
```diff
@@ -974,4 +974,80 @@ curl "https://cloudmc_endpoint/api/v1/organizations/87895f43-51c1-43cc-b987-7e30
...
+`discounts`<br/>*Array[BillingCycle.BillingCycleDiscounts]* | List of discounts applied during the billing cycle.
+`discounts.endDate`<br/>*date* | The end date of the discount.
+`discounts.startDate`<br/>*date* | The start date of the discount.
+`discounts.name`<br/>*Map[String, String]* | The name translations of the discount.
+`discounts.type`<br/>*enum* | The type of the discount. It could be either "PERCENTAGE" or "CREDIT".
+`discounts.remaining`<br/>*BillingCycle.CreditTracking* | Tracking of the credits remaining and used for the billing cycle.
```
Comment (GuillaumeVialle) [inline source/includes/administration/_organizations.md]: same here, object

File: source/includes/administration/_organizations.md [hunk 8]
```diff
@@ -974,4 +974,80 @@ curl "https://cloudmc_endpoint/api/v1/organizations/87895f43-51c1-43cc-b987-7e30
...
+}
+```
+
+Attributes | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The id of the billing cycle.
+`startDate`<br/>*string* | The start date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) of the billing cyle.
+`endDate`<br/>*string* | The end date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) of the billing cyle.
+`currency`<br/>*string* | The currency associated to the organization.
+`deferred`<br/>*boolean* | Indicated whether the billing cycle is deferred or not.
```
Comment (GuillaumeVialle) [inline source/includes/administration/_organizations.md]: Indicates

File: source/includes/administration/_organizations.md [hunk 9]
```diff
@@ -974,4 +974,80 @@ curl "https://cloudmc_endpoint/api/v1/organizations/87895f43-51c1-43cc-b987-7e30
...
-`billingEmails`<br/>*map* | A map of billing emails and names associated with them. There is a maximum of three emails that can be configured. Each entry must have a valid email specified along with a name.
\ No newline at end of file
+`billingEmails`<br/>*map* | A map of billing emails and names associated with them. There is a maximum of three emails that can be configured. Each entry must have a valid email specified along with a name.
+
+<!-------------------- FIND BILLING CYCLE -------------------->
+### Get a billing cycle for an org
+
+`GET /organizations/:organization_id/billing_cycles/:billing_cycle_id`
+
+Retrieves the billing cycle for the organization.
```
Comment (GuillaumeVialle) [inline source/includes/administration/_organizations.md]: Retrieves a billing cycle for the organization.
(multiple billing cycles, we fetch one here)

File: source/includes/administration/_organizations.md (hunk 5 above)
Comment (fyrnaga) [reply source/includes/administration/_organizations.md]: are we returning the complete invoice object or just the id?

Comment (HassanHaidar) [reply source/includes/administration/_organizations.md]: Just IDs

Comment (HassanHaidar) [reply source/includes/administration/_organizations.md]: I guess I can change it to Array[UUID] in this case

Comment (HassanHaidar) [reply source/includes/administration/_organizations.md]: 4f8cda86ce0fcb9055e399d1e28e37c7502d4cd9

File: source/includes/administration/_organizations.md (hunk 6 above)
Comment (HassanHaidar) [reply source/includes/administration/_organizations.md]: 4f8cda86ce0fcb9055e399d1e28e37c7502d4cd9

File: source/includes/administration/_organizations.md (hunk 7 above)
Comment (HassanHaidar) [reply source/includes/administration/_organizations.md]: 4f8cda86ce0fcb9055e399d1e28e37c7502d4cd9

File: source/includes/administration/_organizations.md (hunk 8 above)
Comment (HassanHaidar) [reply source/includes/administration/_organizations.md]: 4f8cda86ce0fcb9055e399d1e28e37c7502d4cd9

File: source/includes/administration/_organizations.md (hunk 9 above)
Comment (HassanHaidar) [reply source/includes/administration/_organizations.md]: 4f8cda86ce0fcb9055e399d1e28e37c7502d4cd9

File: source/includes/administration/_organizations.md (hunk 2 above)
Comment (HassanHaidar) [reply source/includes/administration/_organizations.md]: 4f8cda86ce0fcb9055e399d1e28e37c7502d4cd9

File: source/includes/administration/_organizations.md (hunk 3 above)
Comment (HassanHaidar) [reply source/includes/administration/_organizations.md]: 4f8cda86ce0fcb9055e399d1e28e37c7502d4cd9

File: source/includes/administration/_organizations.md (hunk 4 above)
Comment (HassanHaidar) [reply source/includes/administration/_organizations.md]: 4f8cda86ce0fcb9055e399d1e28e37c7502d4cd9

**Review by fyrnaga — APPROVED** [review]

**Review by GuillaumeVialle — APPROVED** [review]
Thanks for the changes

**Pull request merged**